site/DataPage.tsx

```diff
@@ -166,6 +169,10 @@ export const DataPage = (props: {
                     context={SiteFooterContext.dataPage}
                     isPreviewing={isPreviewing}
                 />
+                <script
+                    type="module"
+                    dangerouslySetInnerHTML={{ __html: script }}
```

Comment:
We're serializing and embedding `grapherConfig` twice on this page now. 

Is this because this PR is an emergency band-aid or should we take a little longer to update this so that we only have to do it once?